### PR TITLE
Guess-who 

### PR DESCRIPTION
**Pull request by jessicatf**

### CODE REVIEW

**Comment by jessicatf** [comment]
...

File: code/script.js [hunk 1]
```diff
@@ -35,15 +42,15 @@ const CHARACTERS = [
...
+    other: ['mouth-open']
   },
   {
     name: 'Jake',
     img: 'images/jake.svg',
     hair: 'yellow',
     eyes: 'green',
     accessories: ['glasses'],
-    other: []
+    other: ['multi-colored-shirt','mouth-open']
```
Comment (amandatilly) [inline code/script.js:53]: Great that you added more options here

File: code/script.js [hunk 2]
```diff
@@ -218,108 +227,180 @@ const generateBoard = () => {
...
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
-}
+  // sets charactersInPlay array to be all the characters to start with
+  
+  charactersInPlay = CHARACTERS;
+  // show the board, activating the secret person and the selected question.
+  countGuess.innerHTML = "Number of guesses : ";
```
Comment (amandatilly) [inline code/script.js:248]: good work adding a counter!

File: code/script.js [hunk 3]
```diff
@@ -218,108 +227,180 @@ const generateBoard = () => {
...
-
-// when clicking guess, the player first have to confirm that they want to make a guess.
-const guess = (personToConfirm) => {
-  // store the interaction from the player in a variable.
-  // remember the confirm() ?
-  // If the player wants to guess, invoke the checkMyGuess function.
-}
+const guess = (personToCheck) => {
+  
+  let playerGuess = confirm(`Do you want to guess on this fellow ${personToCheck}?`);
```
Comment (amandatilly) [inline code/script.js:368]: I liked that you added the name of the person you're guessing on here.

File: code/script.js [hunk 4]
```diff
@@ -218,108 +227,180 @@ const generateBoard = () => {
...
 const checkMyGuess = (personToCheck) => {
-  // 1. Check if the personToCheck is the same as the secret person's name
-  // 2. Set a Message to show in the win or lose section accordingly
-  // 3. Show the win or lose section
-  // 4. Hide the game board
-}
+  if (personToCheck === secret.name) {
+    winOrLoseText.innerHTML = `You won! You guessed on the correct person - ${personToCheck}!`;
+  } else {
+    winOrLoseText.innerHTML = ` Bad guess! this was not the right person. It was ${secret.name}.`;
```
Comment (amandatilly) [inline code/script.js:382]: also here that it displays the person who was the secret one when you guessed wrong. Nice!

**Review by amandatilly — COMMENTED** [review]
Great job! Your game works really well and you have added some great final touches to it like the counter and details like displaying the person's name when guessing and on the win and lose pages. It is responsive and covers the requirements for this week's project.

The styling also looks nice & you should be really proud!

**Pull request closed without merging**